cmd/util-db/db/metadata.go

```diff
@@ -174,7 +174,7 @@ func insertMetadata(ctx *cli.Context) error {
 			return err
 		}
 	case utils.TypePrefix:
-		num, err := strconv.Atoi(valArg)
+		num, err := strconv.ParseUint(valArg, 10, 8)
```

Comment:
Think about using `strconv.ParseInt()` and using int64 instead of parsing Uint64 into Int32.

Reply:
Not really sure if you're marking the right place?
We are trying to get a byte from string, so I'm parsing with Uint8.

Reply:
I see your point. The underlying type of the return int is always uint64 when using `ParseUInt`.

Reply:
I think I see your point. The result of `strconv.ParseUInt()` is Uint64, so we should convert this to Uint8 before giving it to byte.